pkg/routeagent_driver/handlers/kubeproxy/endpoint_handler.go

```diff
@@ -101,6 +101,15 @@ func (kp *SyncHandler) RemoteEndpointCreated(endpoint *submV1.Endpoint) error {
 
 	kp.syncHandlerMutex.Lock()
 	defer kp.syncHandlerMutex.Unlock()
+
+	lastPorcessedTime, ok := kp.remoteEndpointTimeStamp[endpoint.Name]
+
+	if ok && lastPorcessedTime.After(endpoint.CreationTimestamp.Time) {
+		klog.Infof("Ingoring new remote %#v since a later endoint was already"+
```

Comment:
I'm not sure how noisy this could be but It seems this should be DEBUG or TRACE level.

Reply:
Ideally, these events should happen only during migration and if there is out-of-order events. So, I would guess it should be less noisy.